### PR TITLE
Feature/stp 949

### PR DESCRIPTION
**Pull request by kakanisridhar**
# Description
This is about adding a phase banner to designer layout to indicate at what development stage form designer is in

- Designer layout changed to include phase banner
- Visualizer and menu styles changed to make menu sticky
- Visualizer getLayout changed in order to accommodate sticky menu and behavior on overfow    

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [X] Manual testing

# Checklist:

- [X] My changes do not introduce any new linting errors
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation and versioning
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have rebased onto master and there are no code conflicts
- [X] I have checked deployments are working in all environments
- [X] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: designer/client/styles/index.scss [hunk 1]
```diff
@@ -82,7 +83,9 @@ main {
 
   @include govuk-media-query($from: tablet) {
     .menu {
-      position: fixed;
+      position: -webkit-sticky; /* Safari */
```
Comment (jonnywyatt) [inline designer/client/styles/index.scss:86]: Better to take care of prefixes in the webpack build? Will ensure they're never missed out 
eg https://github.com/postcss/autoprefixer#webpack

**Review by jenbutongit — APPROVED** [review]
LGTM! 🚀

File: designer/server/__tests__/view.tests.ts [hunk 2]
```diff
@@ -60,4 +68,39 @@ suite("Server Help Pages", () => {
...
+
+    const options = {
+      method: "get",
+      url: "/new",
+    };
+
+    const res = await server.inject(options);
+    expect(res.statusCode).to.equal(200);
+    expect(
+      res.result.indexOf(
```
Comment (jonnywyatt) [inline designer/server/__tests__/view.tests.ts:86]: Just a thought, but Cheerio can make this kind of thing easier (will parse the server text response into a queryable DOM object)

File: designer/server/views/designer.html [hunk 3]
```diff
@@ -9,6 +9,19 @@
...
 
 {% block content %}
+    {% if (phase) %}
+    <div class="govuk-phase-banner">
+      <p class="govuk-phase-banner__content">
+        <strong class="govuk-tag govuk-phase-banner__content__tag">
+        {{ phase }}
+        </strong>
+        <span class="govuk-phase-banner__text">
+        This service is currently in development
```
Comment (jonnywyatt) [inline designer/server/views/designer.html:19]: i18n string here?

Comment (jenbutongit) [reply designer/server/views/designer.html:19]: this is one of the njk templates.. we don't have i18n hooked up to this (if at all)..!

Comment (jonnywyatt) [reply designer/server/views/designer.html:19]: 👍

File: designer/client/styles/index.scss (hunk 1 above)
Comment (kakanisridhar) [reply designer/client/styles/index.scss:86]: yes, i will bring in postcss

File: designer/server/__tests__/view.tests.ts (hunk 2 above)
Comment (kakanisridhar) [reply designer/server/__tests__/view.tests.ts:86]: Smoke tests are added by carlos , I am not sure how long will this banner last, don't want to bring in another dep

File: designer/package.json [hunk 4]
```diff
@@ -6,6 +6,10 @@
   "engines": {
     "node": ">=12"
   },
+  "browserslist": [
```
Comment (jonnywyatt) [inline designer/package.json:9]: There's a `.browserslistrc` in the root folder of the repo already which is used by babel-preset-env; if switching to this, pls delete it

**Comment by TheSpartan1980** [comment]
@kakanisridhar I'm happy from a QA standpoint

File: designer/package.json (hunk 4 above)
Comment (kakanisridhar) [reply designer/package.json:9]: ok i will remove this section

**Review by TheSpartan1980 — APPROVED** [review]

**Review by jenbutongit — APPROVED** [review]

**Pull request merged**